docs/reference/ilm/error-handling.asciidoc

```diff
@@ -211,4 +213,4 @@ For more information, see <<circuit-breaker-errors>>.
 This indicates that the cluster is running out of disk space.
 This can happen when you don't have {ilm} set up to roll over from hot to warm nodes.
 
-Consider adding nodes, upgrading your hardware, or deleting unneeded indices.
\ No newline at end of file
+Consider adding nodes, upgrading your hardware, or deleting unneeded indices.
```

Comment:
is this a change?

Reply:
It doesn't look like it, I will check what happened there